src/Domain/ScsPallet.cs

```diff
@@ -1,4 +1,4 @@
-﻿namespace Linn.Stores.Domain
+﻿namespace Linn.Stores.Domain.LinnApps.Scs
```

Comment:
should namespace match file location?

Reply:
files got moved about a bit - thought I'd caught them all will fix